client/src/hooks/networks/index.ts

```diff
@@ -379,6 +371,8 @@ const useGetNetworksRelations = ({ id, type }: Network) => {
   } = useFunction(
     id as number,
     {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
There should be a way to fix this on the `cms/config/plugins.ts` file where we override the wrong strapi types. If not fixed in this PR maybe we can add a new bug to Jira to fix it at some point

Reply:
@Bluesmile82 Have you done it in the past? If so, could you help me?